internal/test/resources.go

```diff
@@ -654,6 +655,19 @@ func (r *TestResources) NewCryostatWithDatabaseSecretProvided() *model.CryostatI
 	return cr
 }
 
+func (r *TestResources) NewCryostatWithAdditionalMetadata() *model.CryostatInstance {
+	cr := r.NewCryostat()
+	cr.Spec.Metadata = &v1beta1.ResourceMetadata{
+		Labels: map[string]string{
+			"myExtraLabel": "myLabel",
+		},
+		Annotations: map[string]string{
+			"myExtraAnnotation": "myAnnotation",
+		},
+	}
```

Comment:
Could I ask you to add some of the default labels and annotations here (e.g. `app`, `component`, `app.openshift.io/connects-to`) with arbitrary values. Then the test case can verify that the operator replaces the values for those labels/annotations with its own values.

Reply:
Added